src/Filter/PipeInputFilter.php

```diff
@@ -0,0 +1,76 @@
+<?php
+/**
+ * MiniAsset
+ * Copyright (c) Mark Story (http://mark-story.com)
+ *
+ * Licensed under The MIT License
+ * For full copyright and license information, please see the LICENSE.txt
+ * Redistributions of files must retain the above copyright notice.
+ *
+ * @copyright     Copyright (c) Mark Story (http://mark-story.com)
+ * @since         0.0.1
+ * @license       http://www.opensource.org/licenses/mit-license.php MIT License
+ */
+namespace MiniAsset\Filter;
+
+use MiniAsset\Filter\AssetFilter;
+use MiniAsset\Filter\CssDependencyTrait;
+
+/**
+ * Pre-processing filter that adds support for command pipes
+ */
+class PipeInputFilter extends AssetFilter
+{
+    use CssDependencyTrait {
+        getDependencies as getCssDependencies;
+    }
+
+    protected $_settings = array(
+        'ext' => '.css',
+        'command' => '/bin/cat',
+        'dependencies' => 'none', // none, css, other
+        'optional_dependency_prefix' => false,
+        'path' => '/bin',
+    );
+
+    /**
+     * It will use prefixed files if they exist.
+     *
+     * @var string
+     */
+    protected $optionalDependencyPrefix = null;
+
+    public function getDependencies(\MiniAsset\AssetTarget $file)
```

Comment:
Could this class be added to the `use` list? That would be more consistent with the rest of this project.